apps/api/src/domain/games/memory-match/queries/__snapshots__/memory-match-queries.fetch-many.e2e.spec.ts.snap

```diff
@@ -1,13 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`when querying for a memory match round: fetch many when the user is unauthenticated when no filters are provided should return all published rounds 1`] = `
+exports[`when querying for a memory match round: fetch many when the user is unauthenticated (public queries) when no filters are provided should return all published rounds 1`] = `
 [
   {
-    "cardbackImageUrl": "http://localhost:1234/awesome-api/resources/mediaItems/download/9b1deb4d-3b7d-4bad-9bdd-2b0d7b100003",
+    "cardbackImageUrl": "http://localhost/:4600/null/resources/mediaItems/download/9b1deb4d-3b7d-4bad-9bdd-2b0d7b100003",
```

Comment:
Why did the `GLOBAL_PREFIX` disappear?

Reply:
Also, why is there a trailing `/` after localhost? If this is generated by the production code (not a test config typo), images won't render on the client. Of course Cypress will catch this later, but let's get it now.

Reply:
See the comment for the `fetch-by-id` test. You need to inject both the `GLOBAL_PREFIX` (using a non-empty string preferably) and `NODE_PORT` in the test config. This ensures the snapshot is independent of the machine the test runs on.